policy_machine.gemspec

```diff
@@ -20,12 +20,11 @@ Gem::Specification.new do |s|
   s.add_dependency('listen')
 
   # Only required in ActiveRecord mode
-  s.add_dependency('activerecord-import', '~> 0.12')
-    # Only required for mysql
-    s.add_dependency('mysql2')
-    # Only required for postgres
-    s.add_dependency('pg', '< 1.0.0')
-    s.add_dependency('activerecord-hierarchical_query', '~> 0.0')
```

Comment:
what was this?

Reply:
https://github.com/take-five/activerecord-hierarchical_query

it's basically a gem that adds DSL-like functions to AR to generate recursive queries.  it is not actually used as they have instead just written out the queries in SQL directly.  perhaps they were trying to use it as some point, but then gave up and forgot to remove it from dependencies and the one `require`.